ipaserver/install/plugins/update_ra_cert_store.py

```diff
@@ -0,0 +1,86 @@
+# Authors:
+#   Alexander Bokovoy <abokovoy@redhat.com>
```

Comment:
If this is a new file, it should have new shorter header (see test_dnssec.py s example), or at least correct year and author